src/arcade/patch/agent/cell/PatchCellContainer.java

```diff
@@ -69,7 +74,8 @@ public PatchCellContainer(
             double volume,
             double height,
             double criticalVolume,
-            double criticalHeight) {
+            double criticalHeight,
+            Bag cycles) {
```

Comment:
Why is this an argument rather than a field? It seems like it could just be a bag specified in `PatchCell`.